pages/project/read/views/index.jsx

```diff
@@ -368,7 +368,7 @@ export default function ProjectLandingPage() {
   const isEditable = model.status === 'active' || model.status === 'inactive';
   const grantedVersion = model.versions.find(v => v.status === 'granted');
 
-  const canChangeLicenceHolder = canUpdate && !openTask && isEditable && (!model.isLegacyStub || (model.isLegacyStub && asruLicensing));
+  const canChangeLicenceHolder = canUpdate && isEditable && (!model.isLegacyStub || (model.isLegacyStub && asruLicensing));
```

Comment:
How come the `openTask` condition has been removed?

Reply:
No problem, I think I see now. It's evaluated as a sub-condition if `canChangeLicenceHolder` is true.